xml/hwbp_registers.xml

```diff
@@ -189,7 +189,12 @@
         <field name="m" bits="6" access="R/W" reset="0">
             When set, enable this trigger in M mode.
         </field>
-        <field name="0" bits="5" access="R" reset="0" />
+        <field name="hit" bits="5" access="R/W" reset="0">
```

Comment:
this seems rather weirdly stuck in the middle, what used to be `enable this trigger in H mode`. I'm not sure what the priv spec folks are planning for the return of `h` mode, but is there nowhere else to put this that is less awkward than between the `m` and `s` bits?